invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/search.less

```diff
@@ -44,3 +44,7 @@
 .ui.link.items > .item:hover {
   cursor: auto !important;
 }
+
+.ui.active.button.request-search-button {
+  cursor: default;
```

Comment:
Why is this button special in the context of the whole theme? I know that you would like to make the user think the button is no longer "clickable" but I think we should think about the big picture here:
if you change it only for this button, it will be inconsistent with the rest of the semantic ui design. I would just leave the cursor as it is defined in semantic, or, if you insist, you should overwrite the whole `ui.active.button` class, but then it will change across the theme.